### ISSUE TITLE
Improve ctr in serps for releasenotes

### ISSUE THREAD

**Issue by rraue** [mozilla/bedrock#5751]
The relesenotes pages get a lot of traffic and are ranking well for “Firefox X” but do not have the expected ctr in the search results for this rankings. 

Proposal: https://docs.google.com/document/d/1VLMJ9VPJowgqc2rxhumbo58xue151cf4n41e8jcSnhQ/edit#

**Comment by thejeffreytaylor** [comment]
We're in contact with Michelle Heubusch, who writes the release notes, and will work with her on this issue.

**Comment by rraue** [comment]
Updating the <title> of all releasenotes to this format: “Firefox X, See All New Features, Updates and Fixes” 

Firefox: “Firefox 60, See All New Features, Updates and Fixes”
iOS: “Firefox 11.0 for iOS , See All New Features, Updates and Fixes”
Android: “Firefox 60.0.0 for Android , See All New Features, Updates and Fixes”
Beta: same as Firefox, as the version includes the beta. “Firefox 61.0beta, See All New Features, Updates and Fixes”
Nightly: “Firefox Nightly 62.0a1, See All New Features, Updates and Fixes”
Android beta: same as firefox beta “Firefox 61.0beta for Android , See All New Features, Updates and Fixes”

**Comment by stephaniehobson** [comment]
@rraue We don't quite have the control we need to implement this request 100% but how's this:

Firefox 61.0.1, See All New Features, Updates and Fixes
Firefox for iOS 12.0, See All New Features, Updates and Fixes
Firefox for Android 61.0, See All New Features, Updates and Fixes
Firefox Beta 62.0beta, See All New Features, Updates and Fixes
Firefox Nightly 62.0a1, See All New Features, Updates and Fixes
Firefox for Android Beta 62.0beta, See All New Features, Updates and Fixes

**Comment by rraue** [comment]
That's much better as well, thank you @stephaniehobson 🦊

**Comment by stephaniehobson** [comment]
This is live :)

**Comment by rraue** [comment]
I updated the brief document with the test results: https://docs.google.com/document/d/1VLMJ9VPJowgqc2rxhumbo58xue151cf4n41e8jcSnhQ/edit#

tldr:
Sessions decreased by 2,5%
Desktop downloads conversion rate increased by 6,9%
Desktop downloads increased by 4,2%
Bounce rate decreased by 13,2%
Session duration increased by 5,5%

I call it succesful 🔝

**Comment by stephaniehobson** [comment]
💯
